GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.ts

```diff
@@ -156,7 +170,70 @@ namespace gdjs {
           );
         }
       };
-      return;
+
+      ((log, warn, error, gdjsLog) => {
+        // Hook the console logging functions to log to the Debugger as well
+        console.log = (...messages) => {
```

Comment:
Can you check how `...rest` arguments are handled by esbuild in the generated code?

Reply:
They are emitted unchanged. I think `...rest` is compatible with a lot of browsers, according to [Can I use](https://caniuse.com/mdn-javascript_operators_spread_spread_in_function_calls) support for spread arguments in functions have been added alongside the [`let`](https://caniuse.com/let) keyword, which we use a lot so we should be safe.

Reply:
Alright, thanks for checking!